pgpool2-4.6.yaml

```diff
@@ -63,6 +68,45 @@ subpackages:
       runtime:
         - ${{package.name}}
 
+  - name: ${{package.name}}-compat
+    description: "Compatibility package for Upstream pgpool image"
+    dependencies:
+      provides:
+        - pgpool2-compat=${{package.full-version}}
+      runtime:
+        - bash
+        - coreutils
+        - busybox
```

Comment:
I'll remove busybox in favor of coreutils, findutils, grep, gnutar, gawk etc.

if coreutils and bash are required for entrypoint script, this should be the runtime dependencies of the entrypoint package.

Reply:
Noted, moving these packages to entrypoint subpackage.